### ISSUE TITLE
J2ME support

### ISSUE THREAD

**Issue by ghost** [ba-sz-at/origamieditor3d#2]
Is it possible port this app using next  framework?
* https://github.com/technicat/FuguJava

**Comment by bagyoni** [comment]
I think you're misunderstanding what that framework does. It doesn't really port anything. It only defines an abstract class for writing simple games, with a very limited set of supported UI components, and it implements *those games* in both J2ME and Swing. It doesn't actually support Swing components.
(From a technical viewpoint, the problem is that the OrigamiEditorUI class does not extend the com.technicat.swing.Application class, and it can't be tweaked to do so, because it already extends the javax.swing.JFrame class. But this leads us back to the same basic problem I described earlier.)
It would be more practical to reimplement Origami Editor 3D itself in J2ME. But I don't want to do that because (1.) I would have to rewrite 5000 lines of code (2.) I would have to rewrite any subsequent change to the GUI code twice.
Truth be told, I have already tried to make alternative versions to this program, such as an Android app or a web application, but I sticked with the desktop version to maximize the number of features. I don't think I will ever give up on it in favor of another platform.
